ee/clickhouse/materialized_columns/test/test_columns.py

```diff
@@ -37,10 +42,16 @@ def recreate_database(self):
         create_clickhouse_tables(0)
 
     def test_get_columns_default(self):
-        self.assertCountEqual(get_materialized_columns("events"), EVENTS_TABLE_DEFAULT_MATERIALIZED_COLUMNS)
+        self.assertCountEqual(
+            [property_name for property_name, _ in get_materialized_columns("events")],
+            EVENTS_TABLE_DEFAULT_MATERIALIZED_COLUMNS,
+        )
         self.assertCountEqual(get_materialized_columns("person"), [])
+        # TODO: Check where this is used and how severely its affected. Since we don't materialise anything else in recordings,
+        # weird to have this here.
         self.assertEqual(
-            get_materialized_columns("session_recording_events"), {"has_full_snapshot": "has_full_snapshot"}
+            get_materialized_columns("session_recording_events"),
```

Comment:
cc: @rcmarron , I don't _think_ this change affects anything, but would be nice if you can double check.

Reply:
Yeah, that shouldn't impact anything. The test is just making sure the column is materialized out, but because it's materialized in a migration, we don't use the `get_materialized_columns` function to access it (we just do it directly).

With that said, seems a bit odd for the column name to be wrong. It should probably return `{("has_full_snapshot", "snapshot_data"): "has_full_snapshot"}`.

Reply:
Yeah, this is the tricky bit. The table column is now inferred from the comment, and when no column is in the comment, it defaults to `properties`, which is the column for both events & persons.

This is the wrench^^, I probs need a migration to alter the comment. It's a known default column that's accessed directly always though, & the only materialised one ever on recording_events, so not doing the migration doesn't hurt. Haven't gotten around to it yet.